security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt

```diff
@@ -227,15 +230,6 @@ end
 policy "policy_rds_instance_list" do
   validate_each $ds_rds_unencrypted_instance_list do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} unencrypted RDS Instances Found in AWS"
-    detail_template <<-EOS
-# List of unencrypted RDS Instances
-| RDS Instance Name | Region | Availability Zone | Secondary Availability Zone | Publicly Accessible | DB Engine Name | DB Instance Status | Delete Protection Enabled | TAGs |
-| ----------------- | ------ | ----------------- | --------------------------- | ------------------- | -------------- | ------------------ | ------------------------- | ---- |
-{{ range data -}}
-| {{.db_instance_identifier}} | {{ .region }} | {{ .availability_zone }} |  {{ .sec_availability_zone }} |{{ .publicly_accessible }} | {{ .db_engine_name }} | {{ .db_instance_status }} | {{.delete_protection}} | {{.tag_key_value}} |
-{{ end -}}
-###### Note: Please refer to 'Enable Delete Action' section in the [README](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_unencrypted) file to perform delete operation.
-    EOS
     escalate $report_unencrypted_RDS_instances
     #Uncomment below five lines (remove '#') to enable Delete action.
     #escalate $delete_unencrypted_RDS_instances_approval
```

Comment:
In this example the UI will support the actions, so you include **delete** action.  There is no need to include instructions to uncomment.  Readme should be updated to include the action and remove instruction to uncomment etc.

Reply:
Done !